data-prepper-plugins/lmdb-prepper-state/src/main/java/com/amazon/dataprepper/plugins/prepper/state/LmdbPrepperState.java

```diff
@@ -147,6 +148,15 @@ public long size() {
         }
     }
 
+    /**
+     * TODO: Implement the sizeInBytes function
+     * @return Size of the prepper state file, in bytes.
+     */
+    @Override
+    public long sizeInBytes() {
+        return 2097152;
```

Comment:
This is not really right. Lets move the db size test to each individual prepper state class test, because those will be different between the implementations. For now, you can return 0 from this.

Reply:
Will do